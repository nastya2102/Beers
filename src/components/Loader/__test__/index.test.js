import React from 'react'
import { render } from '@testing-library/react'
import Loader from '../index'

describe('Loader', () => {
  test('should render Loader component', () => {
    const { container } = render(<Loader />)

    expect(container).toBeTruthy()
  })

  test('should match snapshot Loader component', () => {
    const { container } = render(<Loader />)

    expect(container).toMatchSnapshot()
  })
})
