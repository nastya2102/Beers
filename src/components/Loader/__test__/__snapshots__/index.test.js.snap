// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match snapshot Loader component 1`] = `
<div>
  <div
    class="sc-gsTCUz eAmSFD"
  >
    <div
      class="sc-bdfBwQ hinZSD"
      height="300px"
    />
  </div>
</div>
`;
